### DISCUSSION TITLE
Pass data to backend

### DISCUSSION THREAD

**Discussion by jenya87** [geoman-io/leaflet-geoman#839] [Questions & Help]
Hello! Thanks for cool functionality!
And question is: how to get coordinates and radius of circle, which I draw on the map with DrawCircle marker and pass them to my backend (laravel php framework, for eaxmple)?

Thanks!

**Comment by Falke-Design** [comment]
The best way is to create geojson of each layer. For circles you need to add manually the radius, else it will only "exported" as point.

Take a look into this issue where I created a jsfiddle for import / export geojson. #605
